### PR TITLE
[autoscaler] Small fixes for local cluster usability

### PR DESCRIPTION
**Pull request by richardliaw**
## What do these changes do?
Enables multiple cluster names.

## Linter

- [x] I've run `scripts/format.sh` to lint the changes in this PR.

### CODE REVIEW

File: python/ray/autoscaler/updater.py [hunk 1]
```diff
@@ -223,18 +229,19 @@ def do_update(self):
...
         self.provider.set_node_tags(self.node_id,
                                     {TAG_RAY_NODE_STATUS: "setting-up"})
-
         m = "{}: Initialization commands completed".format(self.node_id)
         with LogTimer("NodeUpdater: {}".format(m)):
             with open("/dev/null", "w") as redirect:
                 for cmd in self.initialization_commands:
-                    self.ssh_cmd(cmd, redirect=redirect)
+                    self.ssh_cmd(
+                        cmd, redirect=None if self.verbose else redirect)
```
Comment (ericl) [inline python/ray/autoscaler/updater.py]: Can we make it verbose by default? It's really bad if we hang without messages, and only a minor nuisance to be verbose.

Comment (richardliaw) [reply python/ray/autoscaler/updater.py]: Done

File: python/ray/autoscaler/updater.py [hunk 2]
```diff
@@ -200,9 +203,12 @@ def sync_file_mounts(self, sync_cmd):
...
                     self.ssh_cmd(
                         "mkdir -p {}".format(os.path.dirname(remote_path)),
-                        redirect=redirect,
+                        redirect=None if self.verbose else redirect,
                     )
-                    sync_cmd(local_path, remote_path, redirect=redirect)
+                    sync_cmd(
+                        local_path,
+                        remote_path,
+                        redirect=None if self.verbose else redirect)
```
Comment (ericl) [inline python/ray/autoscaler/updater.py]: Always verbose

**Comment by richardliaw** [comment]
jenkins retest this please

**Review by ericl — APPROVED** [review]

**Pull request merged**